### PR TITLE
adds documentation for did:webs security characteristics

### PR DESCRIPTION
**Pull request by 2byrds**
A follow-up to the conversations we've had about what security characteristics should define did:webs.

### CODE REVIEW

**Review by dhh1128 — APPROVED** [review]

**Comment by dhh1128** [comment]
I am ready to merge this. I will do so in a couple days if there are no objections.

File: spec/security_characteristics.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+#### [[def: KRAM]]: Low Security
+
+KERI Request Authentication Mechanism (KRAM) is the lowest security requirement and can only be used for ephemeral query/reply mechanisms that protect against replay attacks and key compromise attacks at the moment, not over time. This is done with [[ref: KRAM]] which is a non-interactive replay attack algorithm that uses a sliding window of date-time stamps and key state (similar to the the tuple as in BADA-RUN) but the date-time is the replier’s not the querier’s. [[ref: KRAM]] is meant to protect a host when providing access to information to a client from replay attacks by a malicious client. It is not meant to protect the information provided by the host. For that we must use [[ref: BADA-RUN]] or KEL backing. Thus, by itself [[ref: KRAM]] is not suitable to protect on-disk storage (see below).
```
Comment (peacekeeper) [inline spec/security_characteristics.md:51]: "a non-interactive replay attack algorithm" ?

Maybe:
"a non-interactive replay safe algorithm" ?
"a non-interactive replay attack protected algorithm" ?

**Review by peacekeeper — APPROVED** [review]
Reading this, I wonder a bit how much of it applies to did:webs specifically, and how much of it applies to underlying KERI. For example, it mentions KRAM, but doesn't say what that means for a did:webs identifier or how it can be used with it. But still, I think this an impressive list of security characteristics that should go it.

**Comment by 2byrds** [comment]
That's a good point @peacekeeper. Sam taught me these concepts as I was exploring the security considerations for did:webs. KRAM is the least useful, so far, since we'll mostly use KEL-backed and possibly BADA-RUN for discovery info... because we are focused on a secure asynchronous architecture that has to protect data at rest. I'm completely flexible to reduce this section to the minimally useful info based on your sense. The best section is no section, if it's not useful enough.

**Pull request merged**